etc/pyfstat-dev.yml

```diff
@@ -4,19 +4,20 @@ channels:
   - conda-forge
 
 dependencies:
-  - python >=3.8
-  - numpy
-  - matplotlib-base >=2.1
-  - scipy
-  - ptemcee
+  - attrs
   - corner
   - dill
-  - tqdm
+  - lalpulsar >=5.0.0
+  - matplotlib-base >=2.1
+  - numpy
   - peakutils
   - pathos
+  - pip
+  - ptemcee
+  - python >=3.8
   - python-lal >=7.1.5
   - python-lalpulsar >=5.0.0
-  - lalpulsar >=5.0.0
-  - pip
+  - scipy
+  - tqdm
   - pip:
     - -e ../[dev]
```

Comment:
There's a problem here: this pulls in all requirements from `setup.py`, including e.g. the monolithic PyPI version of `lalsuite`, so we end up with both conda- and pip-installed versions of lalsuite, which is a mess.

Adding `--no-dependencies` would avoid that, but then we won't pull in the `[dev]` dependencies either. I think the safest is to do that, but add the dev stuff manually here, even if that means double the maintenance in case of updates.

Reply:
Done that for now, but should really also go and check how many of them can be upgraded to conda dependencies instead of pip.

Reply:
will try @Rodrigo-Tenorio 's suggestion of adding a [dev-nolalsuite] instead